Control/DeepSeq.hs

```diff
@@ -105,6 +114,10 @@ import GHC.Stack ( CallStack(..) )
 import GHC.SrcLoc ( SrcLoc(..) )
 #endif
 
+#if __GLASGOW_HASKELL__ >= 708
```

Comment:
The `#if __GLASGOW_HASKELL__ >= 708` checks should be changed to `#if MIN_VERSION_base(4,7,0)`, since it's just guarding the use of `Data.Coerce.coerce` from `base`.